Watchman.DomainModel/Mute/Commands/MarkMuteEventAsUnmutedCommand.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Watchman.Cqrs;
+
+namespace Watchman.DomainModel.Mute.Commands
+{
+    public class MarkMuteEventAsUnmutedCommand : ICommand
+    {
+        public MuteEvent MuteEvent { get; private set; }
```

Comment:
tutaj lepiej wrzucać po prostu dane, a nie obiekty 
komenda powinna mieć informacje co zmienić na co 

czasami sie nie da, ale jeśli sie da to tak róbmy

Reply:
Czyli samo muteEventId?